crates/shim/README.md

```diff
@@ -66,26 +66,28 @@ impl shim::Task for Service {
 }
 
 fn main() {
-    shim::run::<Service>("io.containerd.empty.v1")
+    shim::run::<Service>("io.containerd.empty.v2")
 }
 
 ```
 
 ## How to use with containerd
 
+**Note**: All operations are in the root directory of `rust-extensions`.
+
 With shim v2 runtime:
 
 ```bash
-$ cargo build --example empty-shim
-$ sudo cp ./target/debug/examples/empty-shim /usr/local/bin/containerd-shim-empty-v2
+$ cargo build --example empty_shim
+$ sudo cp ./target/debug/examples/empty_shim /usr/local/bin/containerd-shim-empty-v2
```

Comment:
/usr/local/bin/containerd-shim-empty-v2 -> /usr/local/bin/containerd-shim-empty-v1?

Reply:
@fuweid I update the sample code because this README uses `containerd-shim-empty-v2` and `io.containerd.empty.v2` in these commands, do you mean that if w keep them consistent, it doesn’t matter whether v1 or v2?

Reply:
yes. it doesn't matter whether v1 or v2. But I would like to keep it with v1 because empty is just released. 
It's just weird that the version starts from 2 not 1....

Reply:
@fuweid I have updated as your suggested that change all `v2` to `v1`